src/components/FormSchemaEditor/index.tsx

```diff
@@ -0,0 +1,151 @@
+import { nanoid } from "nanoid";
+import dynamic from "next/dynamic";
+import React, { Fragment } from "react";
+import { DragDropContext, Droppable, DropResult } from "react-beautiful-dnd";
+import { Button } from "../atomic";
+import FormQuestionSchemaEditor from "./FormQuestionSchemaEditor";
+import { getUpdateFunction, reindexItemInList } from "./utils";
+
+const NoSSRComponent = (props: any) => (
+  <React.Fragment>{props.children}</React.Fragment>
+);
+
+const NoSSR = dynamic(() => Promise.resolve(NoSSRComponent), {
+  ssr: false,
+});
+
+// Types
+
+export type RecursivePartial<T> = {
```

Comment:
Does this end up being used anywhere besides in utils.tsx?

Reply:
Probably, but we can move the definition somewhere better in the future I think